modules/openssl-nginx/templates/cert.cnf.erb

```diff
@@ -44,6 +44,7 @@ extendedKeyUsage = "<%= @extkeyusage.collect! {|x| "#{x}" }.join(', ') -%>"
 <% end -%>
 
 [ alt_names ]
+DNS.0 = localhost
```

Comment:
Didn't realise `0` was supported. TIL

Reply:
Yeah, I went with updating the numbers below to 2 and 3, but then found 0 just works!